data/data.json

```diff
@@ -10292,6 +10292,49 @@
    "aciGbpServerContainer": "noiro/gbp-server:5.2.3.2.1d150da",
    "aciOpflexServerContainer": "noiro/opflex-server:5.2.3.2.1d150da"
   },
+  "v1.22.16-rancher1-1": {
```

Comment:
There probably won't be 1.22, see more info here: https://github.com/rancher/hyperkube/pull/291#issuecomment-1308896775

Reply:
Disregard, 1.22.16 actually got released (https://github.com/kubernetes/kubernetes/releases/tag/v1.22.16)